### DISCUSSION TITLE
Can't get SharePoint Administrators Group using Set-SPOUser

### DISCUSSION THREAD

**Discussion by KNT2713** [pnp/powershell#2637] [Q&A]
I want to get SharePoint Administrators Group using Get-SPOUser.

I verified it. As a result, I was able to get SharePoint Administrators Group in a certain Tenant.
But I was not able to get SharePoint Administrators Group in a other Tenant.
"https://XXX-admin.sharepoint.com/" is set to site parameter of Get-SPOUser.

Do you know why there is a cause?

**Comment by AndersRask** [comment]
Wrong framework.
This repo is for PnP.PowerShell not Microsoft.Online.SharePoint.PowerShell that are the module with SPO cmdlets

**Reply by KNT2713** [reply]
Sorry.
I was guessing.
I'll ask the question elsewhere.
